.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!-- This is a pull request template, you do not need to uncomment or remove the comments, they won't show up in the PR text. -->
+
+<!-- Your Pull Request name should start with one of the following tags
+  new: For new features
+  improve: For an improvement (performance or little improvements) in existing features
+  fix: For bug fixes that affects the end user
+  break: For pull requests including breaking changes
+  chore: For small tasks
+  doc: For documentation
+-->
+
+<!-- Checklist!!! If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code. -->
+
+- [x] I have read the Contributing Guide
+- [x] I have signed the CLA
+- [x] Lint and unit tests pass locally with my changes
+- [ ] I have labeled the PR correctly with the related package
+- [ ] I have run Loki's visual regression tests (if applicable)
+- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
+- [ ] I have added necessary documentation (if applicable)
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
I'm not sure this is necessary.

Reply:
@tassoevan This one is just a proposal. The thought is that as an open project, we're receiving pull requests from the community, so I think could be good to have a kind of pattern, maybe?